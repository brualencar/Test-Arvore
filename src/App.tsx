import React from 'react';

import './App.css';

function App() {
  return <div>New App</div>;
}

export default App;
