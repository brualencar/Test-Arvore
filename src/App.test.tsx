import React from 'react';

import { render, screen } from '@testing-library/react';

import App from './App';

describe('renders learn react link', () => {
  it('snapshot', () => {
    render(<App />);
    const linkElement = screen.getByText('New App');
    expect(linkElement).toMatchSnapshot();
  });

  it('should render', () => {
    render(<App />);
    const linkElement = screen.getByText('New App');
    expect(linkElement).toBeTruthy();
  });
});
