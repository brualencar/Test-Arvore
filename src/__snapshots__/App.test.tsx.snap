// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link snapshot 1`] = `
<div>
  New App
</div>
`;
